environments/_users/data_team.yml

```diff
@@ -3,7 +3,6 @@ dev_users:
     - bderenzi
     - mtheis
     - esu
-    - ymsosa
     - slee
     - kchen
   absent: []
```

Comment:
you should add him to the `absent` list which will actually remove his key from the server